app/helpers/turbo/frames_helper.rb

```diff
@@ -42,6 +42,6 @@ def turbo_frame_tag(*ids, src: nil, target: nil, **attributes, &block)
     id = ids.first.respond_to?(:to_key) ? ActionView::RecordIdentifier.dom_id(*ids) : ids.join('_')
     src = url_for(src) if src.present?
 
-    tag.turbo_frame(**attributes.merge(id: id, src: src, target: target).compact, &block)
+    tag_builder.tag_string(:turbo_frame, **attributes.merge(id: id, src: src, target: target).compact, &block)
```

Comment:
The [ActionView::Helpers::TagHelper::TagBuilder](https://github.com/rails/rails/blob/6b2a9f1213d7aa4f9a25db7640ad68ec0d8c31ce/actionview/lib/action_view/helpers/tag_helper.rb#L47) is marked as `:nodoc:`, so it isn't intended to be part of the public API.

If performance is a concern, is there an opportunity to replace `tag.turbo_frame` with [content_tag](https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag):

```suggestion
    content_tag(:turbo_frame, nil, **attributes.merge(id: id, src: src, target: target).compact, &block)
```